src/PowerShellEditorServices.Host/EditorServicesHost.cs

```diff
@@ -231,7 +217,7 @@ public void StartLogging(string logFilePath, LogLevel logLevel)
             await this.editorSession.PowerShellContext.ImportCommandsModule(
                 Path.Combine(
                     Path.GetDirectoryName(this.GetType().GetTypeInfo().Assembly.Location),
-                    @"..\..\Commands"));
+                    @"..\Commands"));

```

Comment:
Here's the command module import @tylerl0706

Reply:
This change should help a lot.  :-)